### PR TITLE
{math}[intel/2019b] ELSI v2.5.0

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1883~~ + ~~#9880~~ (`MUMPS` dep for `PETSc`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/4d192c4b9f546b7a211b45f579a8b58f for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/68ee6cdb2c34cfa3a6c42f7aad6148bf for a full test report.

**Comment by migueldiascosta** [comment]
the variants with `SLEPc` require `PETSc` rebuilt with ~~https://github.com/easybuilders/easybuild-easyconfigs/pull/9880~~, otherwise tests will fail

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/0cbd994e783f08ca23c558f7d6989bf0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
node2403.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7a9e644615025a4fa9d06b881a8037a8 for a full test report.

File: easybuild/easyconfigs/e/ELSI/ELSI-2.5.0-foss-2019b_slepc.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name = 'ELSI'
+version = '2.5.0'
+versionsuffix = '_slepc'
```
Comment (boegel) [inline easybuild/easyconfigs/e/ELSI/ELSI-2.5.0-foss-2019b_slepc.eb]: @migueldiascosta I would use `-SLEPc` instead?

How exactly is an `ELSI` installation that depends on `SLEPc` different? It's not just an enhanced installation compared to without `SLEPc` dep?

Comment (migueldiascosta) [reply easybuild/easyconfigs/e/ELSI/ELSI-2.5.0-foss-2019b_slepc.eb]: There's a dependency conflict between the internal `PEXSI` and `SLEPc`, hence the variants. We could make the one with `SLEPc` the default, though...

**Comment by migueldiascosta** [comment]
I can't reproduce the `foss` failures... :(

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/cbd935065670f44486bc0fa5d538c69b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3118.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/287dac55bbaec9d79181436175a28407 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2419.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d201e75492fc208906481883a2a9b329 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @migueldiascosta!

**Pull request merged**